src/mlpack/bindings/python/mlpack/pythonize_model.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+#
+# Utilities for model parameters extraction.
+#
+# @file pythonize_model.py
+# @author Mehul Kumar Nirala
+#
+# mlpack is free software; you may redistribute it and/or modify it under the
+# terms of the 3-clause BSD license.  You should have received a copy of the
+# 3-clause BSD license along with mlpack.  If not, see
+# http://www.opensource.org/licenses/BSD-3-Clause for more information.
+
+
+import numpy as np
+from collections import defaultdict
+from xml.etree import cElementTree as ET
+
+def etree_to_dict(t, attrib = False):
+    """
+    Convert ElementTree to python dict.
+    """
```

Comment:
Looks like this file is done with four spaces per tab, but can we do it to match the rest of the mlpack style instead?

Reply:
Apologies for the delay. I 'll do the needful.